packages/kbn-user-profile-components/src/user_profiles_popover.test.tsx

```diff
@@ -121,4 +123,25 @@ describe('UserProfilesPopover', () => {
       </EuiPopover>
     `);
   });
+
+  it('should set `initialFocus` and `searchInputId` props correctly', async () => {
```

Comment:
Do you think we could add a test that ensures that the input box has focus when the popover opens?

Reply:
We'd need to write an end-to-end browser test for that. I think in this case it is acceptable to rely on EUI having correctly tested functionality of the `initialFocus` prop.